tests/series/test_arithmetic.py

```diff
@@ -339,5 +339,5 @@ def test_mod_series(l_dtype, r_dtype) -> None:
 
     mod = left % right
     assert mod.name() == left.name()
-    assert mod.datatype()._is_integer()
+    assert mod.datatype().is_integer()
```

Comment:
`isinstance(mod.datatype(), IntegerType)` or Integral depending on if you want to disambiguate from `integer<->int32` but tbh `IntegerType` is fine